### DISCUSSION TITLE
Account Setup | Help Needed

### DISCUSSION THREAD

**Discussion by TheDarthFather** [firefly-iii/firefly-iii#8985] [Firefly III] [answered]
Hello,
I see there are a few types of accounts to setup (Revenue, Expense, Asset etc) and I am trying to understand how to set up in my situation:
- Both me and my wife are getting the income in the same bank & account ("Main"). 
- We do have 2 credit cards linked to the same account (they are working like debit cards, as here real CC are not existent). 
- We do have 2 Revolut accounts (one each) linked to our phones. (Historical reasons as the local bank were not ready for phone payments at that time). We move money from "Main" to each Revolut based on the needs ...
- I have a savings account (same bank), a retirement investement account (local pension plan), and some investments in stocks (IBKR). I move money monthly from "Main" to investments account. Sometimes to savings, but rarely as this is just the emergency fond ....
- I pull all transactions to from above (main account,  savings account, 2 revoluts) into a online personal finance app, but I would like to extract from there and pull all to Firefly.  

The big question is how to setup the firefly accounts,  as I am a bit lot in the definitions ?? 
(Income & expense is the same for us ... Revolut ones are extra expense ... etc)!??

PS: The CSV file from the online personal finance app can pull the following columns of data:
- Date 
- Details
- Category
- Amount
- Notes: Same as details, but with Credit/Debit in front of the text (not that we need, as amount has +/-)
- Account (Main, Revolut1, Revolut 2)

Thank you!

**Comment by digitlength** [comment, marked as answer]
What you describe - to me - are all asset accounts. 

Note that Firefly is not designed to deal with investment accounts, so your pension and your stock accounts wouldn't normally go into Firefly. You could try Ghostfolio instead for those. Or keep them in Firefly and accept that some reports/dashboards might show slightly funky results depending on how you account for profit/loss in those accounts.

**Reply by TheDarthFather** [reply]
Thanks! I will test starting from here. 
I had the (wrong) impression I need to do some dummy accounts as Revenue account.

PS: On investments & pension I will just put the current value into one account and update from time to time just to have a full picture of the value.

**Comment by digitlength** [comment]
https://docs.firefly-iii.org/references/firefly-iii/account-types/
